specs/protocol/isthmus/exec-engine.md

```diff
@@ -208,17 +212,56 @@ and the `operatorFeeConstant`.
 
 ### Operator Fee
 
-The operator fee, is set as follows:
+The operator fee is integrated directly into the EVM, alongside the standard gas fee and the OP Stack specific L1 data
+fee. This fee follows the same semantics of existing fees charged in the EVM, just with a new fee beneficiary account.
```

Comment:
It would be useful to have a link or description of what you mean by same semantics. That context may not be obvious to all readers

Reply:
This is described below, but we can link to the relevant section of the yellow paper. I'm unaware of any other good plain-english descriptions of the semantics, unless you have any suggestions 😄 

relevant section of the yellow paper is on pg. 8, under section 5: "_Gas and Payment_": https://ethereum.github.io/yellowpaper/paper.pdf

Reply:
There's also the https://github.com/ethereum/execution-specs, if you feel a python impl is better to link to.

Reply:
Linking to the yellow paper would be great